packages/orchestration/src/exos/ica-account-kit.js

```diff
@@ -174,13 +187,23 @@ export const prepareIcaAccountKit = (zone, { watch, asVow }) =>
           });
         },
         /**
+         * This handler fires if the connection is closed due to external
+         * factors (e.g. a packet timeout). It will not fire if a holder calls
+         * `.close()`.
```

Comment:
can we change the name? `onClose` sure sounds like it would be called on `.close()`!